packages/core/src/resolve.ts

```diff
@@ -289,6 +297,14 @@ export async function resolveDocument(opts: {
         return;
       }
 
+      // handle externalValue with relative reference in examples object
+      if (node.externalValue && isRelativeReference(node.externalValue)) {
```

Comment:
Wouldn't it be easier to treat `externalValue` as `$ref` in [isRef](https://github.com/Redocly/redocly-cli/blob/1f0ca52ce083f21bc1b39ec4f5da8a18ece5daf1/packages/core/src/ref-utils.ts#L11)?